compiler/vmgen.nim

```diff
@@ -726,6 +726,9 @@ proc genBinaryABCD(c: PCtx; n: PNode; dest: var TDest; opc: TOpcode) =
   c.freeTemp(tmp2)
   c.freeTemp(tmp3)
 
+template sizeOfLikeMsg(name): string =
+  "'$1' requires `importc` types to be {.completeStruct.} " % [name]
```

Comment:
mixture of three different quoting styles. I mean it is already pretty messed up, but here you use backtick quotes, single quotes, and `{.` quotes.

Reply:
fixed by using only `'` ; I'm all for conventions so this really should be specified in https://nim-lang.org/docs/nep1.html. Here are sane conventions:
* always use single quotes eg `'foo'`
* pragmas should always use a dot, eg: `'.importc'`

rationale:
* `backtick` is not good because some symbols (eg operators) already require a backtick and docgen needs backtick too.
* `'.importc'` is less verbose than `{.importc.}` and equally unambiguous

Reply:
I agree, quoting rules should be specified. I don't really care if it is `backtick`, 'single quote', ´forward quote´, ‚single bottom top‘,  „double buttom top“, ‘single top’, “double top”, »french style«, or whatever. But I think specifying it is only half of the rent. It should be checked by a style check as well. And if it is a requirement to check the style automatically, then it is also important to ask the question of, what style can be checked automatically? I think 'single quotes' can be a problem as they can be ambiguous with other usages of that character.